### ISSUE TITLE
error TS1192: Module "vike-vue/dist/+config"' has no default export

### ISSUE THREAD

**Issue by TimJohns** [vikejs/vike-vue#194]
After updating a number of package dependencies this morning, I started getting the following:

```sh
src/pages/+config.ts:2:8 - error TS1192: Module '"/Users/tjohns/Src/epic-roadtrip-planner-webapp/node_modules/vike-vue/dist/+config"' has no default export.

2 import vikeVue from 'vike-vue/config'
         ~~~~~~~


Found 1 error in src/pages/+config.ts:2

```

I conjecture this is probably due to whatever TypeScript or ESLint change also resulted in https://github.com/vikejs/vike/issues/1838

I'm not a modules expert, so not sure of side effects, but I suspect changing `export { config }` to `export { config as default}` may resolve the specific issue, and indeed changing it by hand in node_modules as a quick check did allow the build to succeed without error.

I have meetings all day, but if I get free I'll try for an actual fix/workaround and if it's good, will submit a PR, but I'm hoping someone beats me to it.

**Comment by brillout** [comment]
I think I see what's going on, let me fix this. (It seems to be a regression I mistakenly introduced.)

**Comment by brillout** [comment]
Released in `0.8.4`.
